src/storage/src/hummock/sstable/mod.rs

```diff
@@ -71,7 +72,7 @@ use crate::store::ReadOptions;
 
 const DEFAULT_META_BUFFER_CAPACITY: usize = 4096;
 const MAGIC: u32 = 0x5785ab73;
-const VERSION: u32 = 1;
+const VERSION: u32 = 2;
```

Comment:
I suggest we maintain a change log on the version here. Each change log item may include the following item:
1. motivation (maybe a issue link, or plain text) 
2. PR number
3. The updated fields in any struct. (type changed, added or removed).

Reply:
> I suggest we maintain a change log on the version here. Each change log item may include the following item:
> 1. motivation (maybe a issue link, or plain text) 
> 2. PR number
> 3. The updated fields in any struct. (type changed, added or removed).

+1